src/application/TestSuite.ts

```diff
@@ -123,12 +131,26 @@ export class TestSuite {
         continue // skip this response
       }
 
+      // List excludeForOperations
+      const optStatusSuccess = this.getTestTypeFromContractTests('statusSuccess')
+      const optResponseTime = this.getTestTypeFromContractTests('responseTime')
+      const optContentType = this.getTestTypeFromContractTests('contentType')
+      const optJsonBody = this.getTestTypeFromContractTests('jsonBody')
+      const optSchemaValidation = this.getTestTypeFromContractTests('schemaValidation')
+      const optHeadersPresent = this.getTestTypeFromContractTests('headersPresent')
+
       // Add status success check
-      if (config.find(({ statusSuccess }) => !!statusSuccess)) {
+      if (
+        config.find(({ statusSuccess }) => !!statusSuccess) &&
+        !inOperations(pmOperation, optStatusSuccess?.excludeForOperations)
```

Comment:
method name is a bit tricky to reason about... can we change to be explicit? `!excludedOperations` or `includedOperations`

Reply:
the method is quite generic, since it checks if a certain operations belongs to an array of operations.